src/js/components/help/application-updates/__snapshots__/mender-deb-package.test.js.snap

```diff
@@ -148,7 +148,15 @@ sudo bash -c 'DEVICE_TYPE="generic-armv6" && \\
 mender setup \\
   --device-type $DEVICE_TYPE \\
   --quiet --demo  && \\
-systemctl restart mender-client'
+systemctl restart mender-client && \\
+(cat &gt; /etc/mender/mender-shell.conf &lt;&lt; EOF
+{
+  "ServerURL": "https://hosted.mender.io",
+  "ShellCommand": "/bin/sh",
```

Comment:
This file is not required so I think we should drop it.

Reply:
I removed the ShellCommand, let's use the default, you are right!